Sources/SwiftInspectorCore/Tests/TypeLocationAnalyzerSpec.swift

```diff
@@ -0,0 +1,334 @@
+// Created by Michael Bachand on 3/28/20.
+//
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Foundation
+import Nimble
+import Quick
+
+@testable import SwiftInspectorCore
+
+final class TypeLocationAnalyzerSpec: QuickSpec {
+
+  override func spec() {
+    var fileURL: URL!
+
+    afterEach {
+      guard let fileURL = fileURL else {
+        return
+      }
+      try? Temporary.removeItem(at: fileURL)
+    }
+
+    describe("analyze(fileURL:)") {
+      context("the type is not present") {
+        let content =
+        """
+        import Foundation
+        """
+        fileURL = try? Temporary.makeFile(content: content)
+
+        it("returns nil") {
+          let sut = TypeLocationAnalyzer(typeName: "Foo")
+          let result = try? sut.analyze(fileURL: fileURL).first
```

Comment:
Should we be checking for an empty array instead here and below where this doesn't find anything?

Reply:
I originally returned a `LocatedType?` and only later made it an array. I didn't clean up as well as I should have https://github.com/fdiaz/SwiftInspector/pull/13/commits/4c1a086b040821e5bee22a6b8313786207fdf7d6